Gruntfile.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(grunt) {
       coverage: {
         src: "src/test/**/*.js",
         options: {
-          istanbulOptions: [ "--include-all-sources" ]
+          includes: [ "src/**/*.js" ]
```

Comment:
When I had tried this before (I think, and maybe I did it wrong), this only shows coverage for the files that you test.  That is, if you don't test a file at all, it won't ding you on the coverage report, which is sortof the exact opposite of what you want.  Maybe I'm mis-remembering that.